olive/passes/openvino/conversion.py

```diff
@@ -19,31 +19,47 @@ class OpenVINOConversion(Pass):
-                type_=List[Tuple],
+                type_=List,
                 required=False,
                 description=(
-                    "Input can be set by passing a list of tuples. "
-                    "Each tuple should contain input name and optionally input type or input shape."
+                    "Set or override shapes for model inputs. "
+                    "It configures dynamic and static dimensions in model inputs "
+                    "depending on your inference requirements."
                 ),
             ),
-            "input_shape": PassConfigParam(
-                type_=List[int],
+            "example_input": PassConfigParam(
+                type_=Any,
                 required=False,
                 description=(
-                    "Input shape(s) that should be fed to an input node(s) of the model."
-                    " Shape is defined as a comma-separated list of integer numbers"
-                    " enclosed in parentheses or square brackets, for example [1,3,227,227]."
+                    "Sample of model input in original framework. "
+                    "For PyTorch it can be torch.Tensor. "
+                    "For Tensorflow it can be tf.Tensor or numpy.ndarray. "
                 ),
             ),
-            "extra_config": PassConfigParam(
+            "compress_to_fp16": PassConfigParam(
+                type_=bool,
+                default_value=True,
+                required=False,
+                description=("Compress weights in output OpenVINO model to FP16. Default is True."),
+            ),
+            "extra_configs": PassConfigParam(
                 type_=Dict,
                 default_value=None,
                 required=False,
                 description=(
-                    "Extra configurations for OpenVINO model conversion. extra_config can be set by"
-                    " passing a dictionary where key is the parameter name, and the value is the parameter value."
-                    " Please check 'mo' command usage instruction for available parameters:"
-                    " https://docs.openvino.ai/latest/openvino_docs_MO_DG_Deep_Learning_Model_Optimizer_DevGuide.html"
+                    "Extra configurations for OpenVINO model conversion. extra_config can be set by "
+                    "passing a dictionary where key is the parameter name, and the value is the parameter value. "
+                    "Please check Conversion Parameters documentation for more details: "
+                    "https://docs.openvino.ai/2023.2/openvino_docs_OV_Converter_UG_Conversion_Options.html"
+                ),
+            ),
+            "output_model ": PassConfigParam(
+                type_=str,
+                default_value="ov_model",
+                required=False,
+                description=(
+                    "Name of the output OpenVINO model. "
+                    "The output model will be saved to the same directory as the input model."
```

Comment:
does look like this is the case based on the code? 

Also maybe a name like `output_file_name` or something would be clearer?

Reply:
This is `output_model` name is same as OpenVINO api parameter name: https://docs.openvino.ai/2023.2/openvino_docs_OV_Converter_UG_Conversion_Options.html, to keep the api parameter names unified. But I forgot to update the description here. Will update it.